Tests/AdyenTests/Adyen Tests/Utilities/ObservableTests.swift

```diff
@@ -82,7 +82,7 @@ class ObservableTests: XCTestCase, Observer {
             XCTAssertEqual(count, 1)
         }
         
-        // Autoreleasepool should have been executed.
+        // Auto release pool should have been executed.
```

Comment:
Minor and not that relevant comment, but it should be "Autorelease pool" -> For reference https://developer.apple.com/documentation/foundation/nsautoreleasepool

Reply:
yes, but `Autorelease` is not a real word -from English stand point-.